app/services/resource_service.py

```diff
@@ -239,6 +239,16 @@ async def fetch_profile_analytics_by_id(id: str) -> Dict:
     return response
 
 
+@retry(attempts=3, delay=1, retry_exceptions=(TooManyRequestException,))
+async def fetch_async_contents_by_id(id: str) -> Dict:
+    url = urllib.parse.urljoin(get_base_url(), f"/v1/social/creators/async/contents/fetch/{id}")
+
+    response: Dict = await invoke_get_url(url=url, headers={}, auth=get_auth())
```

Comment:
would this return all data at once?

Reply:
Yes, this would return the response including id, status, data etc

Reply:
all data matlab, all contents bro... we discussed and resolved this yesterday